ArcBicepTemplate/FullAzureMonitorMetricsProfileParameters.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "azureMonitorWorkspaceResourceId": {
+      "value": "/subscriptions/b9842c7c-1a38-4385-8f39-a51314758bcf/resourceGroups/sohamtestbiceparc/providers/microsoft.monitor/accounts/shamtestbiceparc"
```

Comment:
Thanks Soham, could we use the same placeholder values between the AKS bicep parameters and the ARC? For example like here: https://github.com/Azure/prometheus-collector/blob/main/AddonBicepTemplate/FullAzureMonitorMetricsProfileParameters.json

Reply:
Thanks @gracewehner for catching this. I just updated this.